compiler+runtime/include/cpp/jank/codegen/llvm_processor.hpp

```diff
@@ -67,6 +67,13 @@ namespace jank::codegen
     eval
   };
 
+  enum class var_root_kind : u8
```

Comment:
This warrants explanation. Also, if it's not used in the header, don't put it there!

Reply:
The second part of this hasn't been addressed. The values of this enum are internal to `llvm_processor`, so we should forward declare the enum in the header and then define it in the cpp file. We do this to keep our headers small, since that impacts both compile-time perf and jank's startup perf.